andela/technites-bn-backend
# #168050022 Travel Admins should be able to create Accommodation facilities

#### What does this PR do?
- Enables a travel admin to add accommodation facilities
#### Description of Task to be completed?
- add accommodation facilities table
- add functionality to ensure a travel admin can add accommodation facilities
- add functionality to enable image upload to cloudinary
#### How should this be manually tested?
- git clone this repo and check out to `ft-accomodation-facility-168050022`
- run `npm i` to install all the dependencies
-  run `sequelize db:migrate:undo:all` then `npm run migrate` to add the accommodations schema
- run the application with `npm run dev`
- register a user in the system
- update their `role_value` in the database manually to a value greater than 3. this is with regard to the user role generation system. ensure their account is verified as well.
- login the user and user the token provided in subsequent operations
- send a POST request to  `/accommodations` in postman via a form-data as shown in the image below:
![image](https://user-images.githubusercontent.com/45405036/65159788-e3696300-da34-11e9-990c-480066aeeb00.png)

#### Any background context you want to provide?
- Roles are manually changed in the database as they are being tackled in another task
- Tests will be delivered in the next PR
#### What are the relevant pivotal tracker stories?
[#168050022 ](https://www.pivotaltracker.com/story/show/168050022)
#### What are the relevant Github Issues?
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/45405036/65133945-d08a6a80-da03-11e9-937c-7ed8face379d.png)
#### Questions:
N/A
